backend/package.json

```diff
@@ -16,6 +16,7 @@
     "express": "^4.17.1",
     "express-jwt": "^5.3.3",
     "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.15",
```

Comment:
I was using a lodash method in a previous commit but then used proper sequelize methods and no longer had to use it. I think we should keep it in the package though because we will surely use it later.

Reply:
Completely agree on leaving it in. Lodash should just be a part of js anyways.

Reply:
Yeah, we will find use for it anyway so it's all good